### PR TITLE
Fix dev-setup script

### PR DESCRIPTION
**Pull request by swapnilgm**
Signed-off-by: Swapnil Mhamane <swapnil.mhamane@sap.com>

**What this PR does / why we need it**:
This PR fixes the link to `namespace-garden-dev.yaml` file where ever referenced.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator

```

### CODE REVIEW

File: docs/deployment/aks.md [hunk 1]
```diff
@@ -423,10 +423,10 @@ In this step we create a namespace in Gardener cluster to keep Shoot
 resource definitions. A `project` in Gardener terminology is simply a
 namespace that holds group of Shoots, during this example we'll deploy
 a single Shoot. (Mind the extra labels defined in
-[example/00-namespace-garden-dev.yaml](../../example/00-namespace-garden-dev.yaml)).
```
Comment (rfranzke) [inline docs/deployment/aks.md:426]: Shouldn't this be `06-namespace-garden-dev`?

File: docs/deployment/aks.md [hunk 2]
```diff
@@ -423,10 +423,10 @@ In this step we create a namespace in Gardener cluster to keep Shoot
...
+[example/05-project-dev.yaml](../../example/05-project-dev.yaml)).
 
 ```
-kubectl apply -f example/00-namespace-garden-dev.yaml
+kubectl apply -f example/05-project-dev.yaml
```
Comment (rfranzke) [inline docs/deployment/aks.md]: Shouldn't this be `06-namespace-garden-dev`?

File: docs/deployment/aks.md (hunk 1 above)
Comment (swapnilgm) [reply docs/deployment/aks.md:426]: Done.

File: docs/deployment/aks.md (hunk 2 above)
Comment (swapnilgm) [reply docs/deployment/aks.md]: Done.

**Review by bergerx — COMMENTED** [review]
It could be me misunderstanding #330 (i'm on vacation and just checked it from my phone) but shouldn't the namespace creations be replaced by project creations for future reference here.

I thought the controller that creates the project from existing namespaces is only kept for backward compatibility, so my expectation is to see only project creations and not manual namespace creation.

**Comment by rfranzke** [comment]
@bergerx You're right, it's no misunderstanding. However, when you set up a dev. environment for the first time you won't have the controller running. This is why we decided to (only in the dev-setup!) create the namespace ourselves for now to avoid that the user has to follow a flow like this:

1. Create some resources
2. Start API server
3. Create some resources
4. Start Controller manager
5. Create some resources

Rather, with the current approach we merge 5 into 3. You're right that the project-namespace controller is only kept for backwards compatibility and once we remove it we have to adapt to this flow anyway, but maybe we can then take the time to generally ease the whole setup a little bit.

**Review by rfranzke — APPROVED** [review]

**Pull request merged**